core/.gas-snapshot

```diff
@@ -1,15 +1,15 @@
 BorrowerGasTest:test_addMargin() (gas: 16092)
-BorrowerGasTest:test_borrow() (gas: 95866)
+BorrowerGasTest:test_borrow() (gas: 95911)
 BorrowerGasTest:test_getUniswapPositions() (gas: 5090)
-BorrowerGasTest:test_modify() (gas: 67354)
-BorrowerGasTest:test_modifyWithAnte() (gas: 73922)
-BorrowerGasTest:test_repay() (gas: 97287)
-BorrowerGasTest:test_uniswapDepositInBorrower() (gas: 293540)
+BorrowerGasTest:test_modify() (gas: 67399)
+BorrowerGasTest:test_modifyWithAnte() (gas: 73967)
+BorrowerGasTest:test_repay() (gas: 97332)
+BorrowerGasTest:test_uniswapDepositInBorrower() (gas: 293542)
 BorrowerGasTest:test_uniswapDepositStandard() (gas: 167271)
-BorrowerGasTest:test_uniswapWithdraw() (gas: 135376)
-BorrowerGasTest:test_withdraw() (gas: 122372)
+BorrowerGasTest:test_uniswapWithdraw() (gas: 135412)
+BorrowerGasTest:test_withdraw() (gas: 122417)
 FactoryGasTest:test_createBorrower() (gas: 155929)
-FactoryGasTest:test_createMarket() (gas: 3657871)
+FactoryGasTest:test_createMarket() (gas: 3450501)
```

Comment:
No core code changed, and we're still using Solidity 0.8.17, so afaict these changes in gas are due to upgrading foundry.